recipes/mtsv/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+cxx_compiler:	
```

Comment:
Please put a comment here explaining why the normal GCC 7 C++ compiler doesn't work for this package.

Reply:
Added comment